packages/nitro-app/app/templating/hbs/helpers/viewlist.js

```diff
@@ -1,17 +1,59 @@
+/**
+ * Viewlist helper, can be used to display lists of views filtered by specific include / exclude terms
+ *
+ * Usage:
+ *
+ * Without parameter, all views will be displayed
+ * {{viewlist}}
+ *
+ * With parameter viewIncludes, all views containing at least one of the terms will be displayed
+ * {{viewlist viewIncludes="<term-1>;<term-2>"}}
```

Comment:
I would prefer using a simpler property for viewIncludes and viewExcludes
What about include & exlude? 
At the same time, one could consider whether semicolon or comma is better suited as separator

Reply:
i will rename the params. Regarding separator, I have no hard feelings, happy to use ","?

Reply:
switched to comma separator